app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -287,6 +286,7 @@ class NewMessageFragment : Fragment() {
             editorManager.setupEditorFormatActionsToggle()
 
             externalsManager.observeExternals(newMessageViewModel.arrivedFromExistingDraft())
+            aiManager.observeEverything()
```

Comment:
Do `observeAiPromptStatus()` and `observeAiFeatureFlagUpdates()` really need to be moved in this later part?

I guess you moved `observeAiOutput()` here so it would only update the body of the mail after the first snapshot has been recorded right?